### DISCUSSION TITLE
Support reading for Older .XLS Files

### DISCUSSION THREAD

**Discussion by mfialko** [exceljs/exceljs#2832] [Q&A]
At the moment, ExcelJS only supports .XLSX and .CSV formats, but it doesn't provide support for the older .XLS format. Unfortunately, in our use case, we still have a lot of legacy files in .XLS format, and this has forced us to rely on another library—namely, xlsx—to handle those older files.

Is there any plan to add .XLS support in the future?

**Comment by MathiasEllegaardRitter** [comment]
export json lib can read xls formats, and then you can just pass json data to exceljs.
